File: stock-check-service/src/main/java/com/chrisbaileydeveloper/stockcheckservice/model/StockCheck.java
package com.chrisbaileydeveloper.stockcheckservice.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import jakarta.persistence.*;

@Entity
@Table
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class StockCheck {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String skuCode;
    private Integer quantity;
}

File: stock-check-service/src/main/java/com/chrisbaileydeveloper/stockcheckservice/model/StockCheck.kt
package com.chrisbaileydeveloper.stockcheckservice.model

import jakarta.persistence.*

@Entity
@Table
data class StockCheck(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long = 0L,
    val skuCode: String = "",
    val quantity: Int = 0
) {
    constructor() : this(0L, "", 0)
}
